Standards/CCSMB-15.md

```diff
@@ -0,0 +1,60 @@
+# _CCSMB-12_: Wide Printer Format and Wide Printer Format Array
```

Comment:
The file is called CSMB-15 but the title is CCSMB-12

Reply:
oopsies